apps/ledger-live-desktop/tests/utils/serve-dummy-app.ts

```diff
@@ -0,0 +1,57 @@
+import handler from "serve-handler";
+import http from "http";
+import path from "path";
+
+export const server = http.createServer((request, response) => {
+  // You pass two more arguments for config and middleware
+  // More details here: https://github.com/vercel/serve-handler#options
+  return handler(request, response, {
```

Comment:
Not sure what is happening here by returning a handler or the `public` attribute. Does using `return` not kill the server?

Reply:
Ok makes more sense: https://www.npmjs.com/package/serve-handler